docs/family/Quickstart.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Quickstart
+weight: 1
+---
+
+# Quickstart
+
+This guide walks through the process to install Upbound Universal Crossplane and install the GCP official provider-family.
+
+To use this official provider-family, install Upbound Universal Crossplane into your Kubernetes cluster, install the `Provider`, apply a `ProviderConfig`, and create a *managed resource* in GCP via Kubernetes.
+
+## Install the Up command-line
+Download and install the Upbound `up` command-line.
+
+```shell
+curl -sL "https://cli.upbound.io" | sh
+mv up /usr/local/bin/
+```
+
+Verify the version of `up` with `up --version`
+
+```shell
+$ up --version
+v0.13.0
+```
+
+_Note_: official provider-families only support `up` command-line versions v0.13.0 or later.(CHECK HERE)
+
+## Install Universal Crossplane
+Install Upbound Universal Crossplane with the Up command-line.
+
+```shell
+$ up uxp install
+UXP 1.9.0-up.3 installed
```

Comment:
Lets update to latest version here.

Also let's add a note of the minimum version required for running family providers.

Reply:
I updated and added a note.